assets/src/components/TableAssignment.js

```diff
@@ -0,0 +1,112 @@
+// modified from https://demos.creative-tim.com/material-dashboard-react/?_ga=2.12819711.913135977.1549993496-494583875.1549993496#/table
+
+import React from 'react'
+import withStyles from '@material-ui/core/styles/withStyles'
+import Table from '@material-ui/core/Table'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import HorizontalBar from '../components/HorizontalBar'
+
+const tableStyle = theme => ({
+  table: {
+    marginBottom: '0',
+    width: '100%',
+    maxWidth: '100%',
+    backgroundColor: 'transparent',
+    borderSpacing: '0',
+    borderCollapse: 'collapse'
+  },
+  tableHeadCell: {
+    color: 'inherit',
+    fontSize: '1em'
+  },
+  tableCell: {
+    verticalAlign: 'top',
+    paddingBottom: '0px'
+  },
+  tableResponsive: {
+    width: '100%',
+    marginTop: theme.spacing.unit * 3,
+    overflowX: 'auto'
+  }
+
+})
+
+function CustomAssignmentTable(props) {
+  const {classes, tableHead, tableData} = props
+
+
+  return (
+    <div className={classes.tableResponsive}>
+      <Table>
+        {tableHead !== undefined ? (
+          <TableHead>
+            <TableRow>
+              {tableHead.map((prop, key) => {
+                return (<TableCell className={classes.tableCell + ' ' + classes.tableHeadCell}
+                                   key={key}>{prop}</TableCell>)
+              })}
+            </TableRow>
+          </TableHead>
+        ) : null}</Table>
+      <Table><TableBody>
+        {
+          tableData.map((week, key) => {
+            return (
+              <TableRow key={key}>
+                <TableCell className={classes.tableCell}
+                           key={key + 'week'}>Week {week.week}</TableCell>
+                <TableCell className={classes.tableCell} key={key + 'weekRest'}>
+                  <Table><TableBody>{
```

Comment:
The nested `<Table>` element is leading to the dis-alignment. But if you don't have the `<Table>` element and instead just `<TableBody>` alignment is preserved between the 2nd column and 3rd column but I see warning regarding. 

> Warning: validateDOMNesting(...): <tbody> cannot appear as a child of <td>.
>     in tbody (created by TableBody)
>     in TableBody (created by WithStyles(TableBody))
>     in WithStyles(TableBody) (at TableAssignment.js:62)
>     in td (created by Context.Consumer)
>     in TableCell (created by WithStyles(TableCell))
>     in WithStyles(TableCell) (at TableAssignment.js:61)
>     in tr (created by Context.Consumer)
>     in TableRow (created by WithStyles(TableRow))
>     in WithStyles(TableRow) (at TableAssignment.js:58)
>     in tbody (created by TableBody)
>     in TableBody (created by WithStyles(TableBody))
>     in WithStyles(TableBody) (at TableAssignment.js:54)
>     in table (created by Table)

But the [issue](https://github.com/facebook/react/issues/5652) here suggest to wrap `<TableBody>` inside `<Table>` that make sense.